distributions/aws/examples/rds-s3/README.md

```diff
@@ -118,15 +118,17 @@ kubectl apply -f https://raw.githubusercontent.com/aws/secrets-store-csi-driver-
 ### 3. Configure Katib
 
 
-1. Configure the `apps/katib/upstream/installs/katib-external-db-with-kubeflow/secrets.env` file with the RDS DB name, RDS endpoint URL, RDS DB port, and RDS DB credentials that were configured when following the steps in Create RDS Instance.
-    - For example, if your database name is `kubeflow`, your endpoint URL is `rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com`, your DB port is `3306`, your DB username is `admin`, and your DB password is `Kubefl0w` your `secrets.env` file should look like:
+1. Configure the secret with the RDS DB name, RDS endpoint URL, RDS DB port, and RDS DB credentials that were configured when following the steps in Create RDS Instance.
+    - For example, if your database name is `kubeflow`, your endpoint URL is `rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com`, your DB port is `3306`, your DB username is `admin`, and your DB password is `Kubefl0w` your secret should look like:
     - **Note:** These are the default values for the database name and credentials in cloudformation template for creating the RDS instance, change these according to the values you used
     - ```
-      KATIB_MYSQL_DB_DATABASE=kubeflow
-      KATIB_MYSQL_DB_HOST=rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com
-      KATIB_MYSQL_DB_PORT=3306
-      DB_USER=admin
-      DB_PASSWORD=Kubefl0w
+      dname=kubeflow
+      host=rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com
+      port=3306
+      username=admin
+      password=Kubefl0w
+
+      aws secretsmanager update-secret  --region $CLUSTER_REGION --secret-id rds-secret --secret-string '{"username":"admin","password":"Kubefl0w","dbname":"kubeflow","host":"rm12abc4krxxxxx.xxxxxxxxxxxx.us-west-2.rds.amazonaws.com","port":"3306"}'
```

Comment:
does the current implementation support using a separate database for katib and KFP?

currently it seems like since both are sharing the same secrets that both will have to use the same RDS instance, but customers may want to scale these seperately

Reply:
Good point if that’s the case I can separate the secrets then

Reply:
That was the case before as well, we are asking user to create one RDS instance in pre-requisites. Lets not change the approach in this PR since it is a 2 way door.

@ryansteakley lets create the secret in the pre-requisites itself. Previously, user had to substitute vars in multiple files and hence we had separate instructions for pipelines and Katib. Now, it can be a single step

Also, can you check if it is possible to configure the rds endpoint for pipelines from the secret as well?

Reply:
remove the unused `.env` files

Reply:
folks two things here.
1. I think its fine for now to keep the same secret (which will ultimate result into same RDS) for both pipeline and Katib.
2. However lets have instruction separete for katib and pipeline in such a way that customer can pick and use one of them or combination. 
3. Why do we want RDS endpoint to be part of secret ?

Reply:
1. I don't think it is necessary to have separate instructions for katib and the pipelines since the one secret will be used for both. If they choose to use one of them or in combination the steps wouldn't be different.
2. It should be possible to configure the rds endpoint for pipelines using secrets since it is using a configmap and creating env vars from there but can change to pull from a secret instead.